big_tests/tests/gdpr_SUITE.erl

```diff
@@ -240,14 +254,17 @@ retrieve_private_xml(Config) ->
 retrieve_inbox(Config) ->
     escalus:fresh_story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
             Body = <<"With spam?">>,
+            BobU = escalus_utils:jid_to_lower(escalus_client:username(Bob)),
+            BobS = escalus_utils:jid_to_lower(escalus_client:server(Bob)),
             escalus:send(Bob, escalus_stanza:chat_to(Alice, Body)),
             Msg = escalus:wait_for_stanza(Alice),
             escalus:assert(is_chat_message, [Body], Msg),
-
-            BobJid = escalus_client:short_jid(Bob),
-            ExpectedHeader = ["jid", "content", "unread_count", "msg_id", "timestamp"],
+            ExpectedHeader = ["jid", "content", "unread_count", "timestamp"],
             ExpectedItems = [
-                             #{ "content" => Body, "jid" => BobJid }
+                             #{ "content" => [{contains, Body}],
+                                "jid" => [{contains, BobS},
+                                          {contains, BobU}],
+                                "unread_count" => "1" }
                             ],
             retrieve_and_validate_personal_data(
               Alice, Config, "inbox", ExpectedHeader, ExpectedItems)
```

Comment:
Can we assert the inbox of `Bob` as well?

Reply:
yes, we can